var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -37,6 +37,8 @@ class Rust(Package):
 
     # Stable versions.
     version("1.81.0", sha256="872448febdff32e50c3c90a7e15f9bb2db131d13c588fe9071b0ed88837ccfa7")
+    version("1.80.0", sha256="e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855")
```

Comment:
I cannot confirm this sha256.  I'm getting `6f606c193f230f6b2cae4576f7b24d50f5f9b25dff11dbf9b22f787d3521d672`.

Reply:
I'm getting the same checksum as @tldahlgren.

Reply:
I'll recheck. It might have been from another version. Sorry for the trouble!